app/uk/gov/hmrc/nationalinsurancerecord/config/AppContext.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 HM Revenue & Customs
+ * Copyright 2021 HM Revenue & Customs
```

Comment:
Suggestion

There are some deprecated methods you could update in here

Reply:
removed unnecessary method